lib/builderUtil.js

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+var util = require('./util');
+var AuthSdkError = require('./errors/AuthSdkError');
+
+function validateConfig(args) {
```

Comment:
While we're validating `issuer` and `url` today, I think the plan is to add additional config checks for the `redirectUri`, `scope`, etc. in the future. Given that this function `validateConfig` is asserting multiple properties, it is a bit confusing that we're returning a scrubbed URL.

I'd suggest either:
a) Modify the existing `args.url` to this new value
b) Update the function name to imply that a URL is being returned through a series of tests